src/neuroconv/tools/roiextractors/roiextractors.py

```diff
@@ -351,12 +355,21 @@ def add_photon_series(
             "OnePhotonSeries" not in metadata_copy["Ophys"]
         ), "Received metadata for 'OnePhotonSeries' but `photon_series_type` was not explicitly specified."
 
+    assert nwbfile_module_name in [
+        "acquisition",
+        "processing",
+    ], "'nwbfile_module_name' must be either 'acquisition' or 'processing'."
+
     # Tests if TwoPhotonSeries//OnePhotonSeries already exists in acquisition
     photon_series_kwargs = metadata_copy["Ophys"][photon_series_type][photon_series_index]
     photon_series_name = photon_series_kwargs["name"]
 
-    if photon_series_name in nwbfile.acquisition:
-        warn(f"{photon_series_name} already on nwbfile")
+    ophys = get_module(nwbfile, name="ophys")
+    if nwbfile_module_name == "acquisition" and photon_series_name in nwbfile.acquisition:
+        warn(f"{photon_series_name} already added to nwbfile.acquisition.")
+        return nwbfile
+    elif nwbfile_module_name == "processing" and photon_series_name in ophys.data_interfaces:
+        warn(f"{photon_series_name} already added to nwbfile.processing['ophys'].")
```

Comment:
Should this warn or error? I would vote for error, since otherwise nothing will happen here

Reply:
I think that's fair, changed to `ValueError` in https://github.com/catalystneuro/neuroconv/commit/480120f57a8d5cb2c63a0eef001b0d86bfb9492f